### PR TITLE
adding some history

### PR DESCRIPTION
**Pull request by ultrasaurus**
update some of the history for https://github.com/cncf/sig-security/issues/414

### CODE REVIEW

File: roadmap.md [hunk 1]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
...
+| Security Reviews/Assessments process improvements | 23 Feb 2021 | [security evaluation process - documentation update](https://github.com/cncf/sig-security/pull/488) |
+| Cloud Native Security Whitepaper | 18 Nov 2020 | [issue#452](https://github.com/cncf/sig-security/issues/452) |
+| First five security assessments | 21 Oct 2020 | [issue#167](https://github.com/cncf/sig-security/issues/167) |
+| First Tech Leads  | 25 Feb 2020 | [TOC approval](https://lists.cncf.io/g/cncf-toc/topic/71341283#4198) |
+| Security Assessment Intake Process | 7 Jan 2020 | [intake process and prioritization](https://github.com/cncf/sig-security/pull/206) |
+| First Cloud Native Security Day | 19 Nov 2019 | [issue#209](https://github.com/cncf/sig-security/issues/209) |
+| Software supply chain catalog  | 14 Nov 2019 | [catalog](https://github.com/cncf/sig-security/pull/284) |
+| Personas & Use cases | 23 Sept 2019 | [issue#246](https://github.com/cncf/sig-security/issues/246)
+| Policy formal verification overview | 10 Sept 2019 | [documentation](https://github.com/cncf/sig-security/pulls/242)
```
Comment (ashutosh-narkar) [inline roadmap.md:50]: `https://github.com/cncf/sig-security/pulls/242` -> `https://github.com/cncf/sig-security/pull/242`

File: roadmap.md [hunk 2]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
...
 |   Milestone  | Date | Action
 | --- | --- | --- |
+| Tech Leads: @ashutosh-narkar, @achetal01, @anvega | 13 Jan 2021 | [TOC Approval](https://lists.cncf.io/g/cncf-toc/topic/79052801#5599) |
+| APAC meetings start | 1 Feb 2021 | [regular meeting time added to README](https://github.com/cncf/sig-security/pull/518)
+| Chinese Translation of White Paper | 27 Feb 2021 | [PR#471](https://github.com/cncf/sig-security/pull/471) |
+| Security Reviews/Assessments process improvements | 23 Feb 2021 | [security evaluation process - documentation update](https://github.com/cncf/sig-security/pull/488) |
+| Cloud Native Security Whitepaper | 18 Nov 2020 | [issue#452](https://github.com/cncf/sig-security/issues/452) |
+| First five security assessments | 21 Oct 2020 | [issue#167](https://github.com/cncf/sig-security/issues/167) |
+| First Tech Leads  | 25 Feb 2020 | [TOC approval](https://lists.cncf.io/g/cncf-toc/topic/71341283#4198) |
+| Security Assessment Intake Process | 7 Jan 2020 | [intake process and prioritization](https://github.com/cncf/sig-security/pull/206) |
```
Comment (ashutosh-narkar) [inline roadmap.md:46]: Can #206 be closed ?

**Review by ashutosh-narkar — APPROVED** [review]
LGTM

File: roadmap.md [hunk 3]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
 
 |   Milestone  | Date | Action
 | --- | --- | --- |
+| Tech Leads: @ashutosh-narkar, @achetal01, @anvega | 13 Jan 2021 | [TOC Approval](https://lists.cncf.io/g/cncf-toc/topic/79052801#5599) |
+| APAC meetings start | 1 Feb 2021 | [regular meeting time added to README](https://github.com/cncf/sig-security/pull/518)
+| Chinese Translation of White Paper | 27 Feb 2021 | [PR#471](https://github.com/cncf/sig-security/pull/471) |
+| Security Reviews/Assessments process improvements | 23 Feb 2021 | [security evaluation process - documentation update](https://github.com/cncf/sig-security/pull/488) |
```
Comment (joshuagl) [inline roadmap.md:42]:
```suggestion
| Chinese Translation of White Paper | 27 Feb 2021 | [PR#471](https://github.com/cncf/sig-security/pull/471) |
| Security Reviews/Assessments process improvements | 23 Feb 2021 | [security evaluation process - documentation update](https://github.com/cncf/sig-security/pull/488) |
| APAC meetings start | 1 Feb 2021 | [regular meeting time added to README](https://github.com/cncf/sig-security/pull/518)
| Tech Leads: @ashutosh-narkar, @achetal01, @anvega | 13 Jan 2021 | [TOC Approval](https://lists.cncf.io/g/cncf-toc/topic/79052801#5599) |
```

Suggest reverse chronological order, like the other milestones in the table.

File: roadmap.md [hunk 4]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
...
+| First five security assessments | 21 Oct 2020 | [issue#167](https://github.com/cncf/sig-security/issues/167) |
+| First Tech Leads  | 25 Feb 2020 | [TOC approval](https://lists.cncf.io/g/cncf-toc/topic/71341283#4198) |
+| Security Assessment Intake Process | 7 Jan 2020 | [intake process and prioritization](https://github.com/cncf/sig-security/pull/206) |
+| First Cloud Native Security Day | 19 Nov 2019 | [issue#209](https://github.com/cncf/sig-security/issues/209) |
+| Software supply chain catalog  | 14 Nov 2019 | [catalog](https://github.com/cncf/sig-security/pull/284) |
+| Personas & Use cases | 23 Sept 2019 | [issue#246](https://github.com/cncf/sig-security/issues/246)
+| Policy formal verification overview | 10 Sept 2019 | [documentation](https://github.com/cncf/sig-security/pulls/242)
+| First Security Assessment | May 2019 | in-toto - [issue#166](https://github.com/cncf/sig-security/issues/166)  |
+| First cut security audit guidelines  | 2 May 2019 | [guidelines](https://github.com/cncf/sig-security/pull/125) |
 | Updated Charter and Governance ratified by CNCF TOC |  7 May 2019 | [New repo](https://github.com/cncf/sig-security/tree/master/governance) |
```
Comment (joshuagl) [inline roadmap.md:53]:
```suggestion
| Updated Charter and Governance ratified by CNCF TOC |  7 May 2019 | [New repo](https://github.com/cncf/sig-security/tree/master/governance) |
| First cut security audit guidelines  | 2 May 2019 | [guidelines](https://github.com/cncf/sig-security/pull/125) |
```
Suggesting reverse chronological sort

File: roadmap.md (hunk 2 above)
Comment (ultrasaurus) [reply roadmap.md:46]: oops -- that link is wrong!   I think I meant https://github.com/cncf/sig-security/pull/296 
I will fix

**Comment by ultrasaurus** [comment]
Realized we're missing Emily as co-chair.   On reflection, I think the roadmap history isn't idea place for the history of who held which role (maybe a history file in governance?).   Related to that, I'm going to work on better words for new items with regard to columns "Milestone" and "Action" -- I like the descriptive words with a link in original list, which means that it could be printed (or displayed on a screen) and still be useful.  Suggestions welcome :)

File: roadmap.md [hunk 5]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
 
 |   Milestone  | Date | Action
 | --- | --- | --- |
+| First Community Translation | 27 Feb 2021 | [Chinese Translation of White Paper](https://github.com/cncf/sig-security/pull/471) |
```
Comment (lumjjb) [inline roadmap.md:39]:
```suggestion
| First Community Translation | 27 Feb 2021 | [Chinese translation of Whitepaper](https://github.com/cncf/sig-security/pull/471) |
```

File: roadmap.md [hunk 6]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
...
+| Security Assessments => Reviews | 23 Feb 2021 | Retrospective resulted in [process updates](https://github.com/cncf/sig-security/pull/488) |
+| APAC meetings start | 1 Feb 2021 | [regular meeting time added to README](https://github.com/cncf/sig-security/pull/518)
```
Comment (lumjjb) [inline roadmap.md:41]:
```suggestion
| APAC meetings start | 1 Feb 2021 | [Regular meeting time added to README](https://github.com/cncf/sig-security/pull/518)
```

File: roadmap.md [hunk 7]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
...
+| Expanded to 5 Tech Leads | 13 Jan 2021 | [TOC Approves](https://lists.cncf.io/g/cncf-toc/topic/79052801#5599) [@ashutosh-narkar](https://github.com/ashutosh-narkar), [@achetal01](https://github.com/achetal01), [@anvega](https://github.com/anvega) |
+| Cloud Native Security Whitepaper v1 | 18 Nov 2020 | [markdown source and images in repo](https://github.com/cncf/sig-security/pull/452) |
```
Comment (lumjjb) [inline roadmap.md:43]:
```suggestion
| Cloud Native Security Whitepaper v1 | 18 Nov 2020 | [Markdown source and images in repo](https://github.com/cncf/sig-security/pull/452) |
```

File: roadmap.md [hunk 8]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
...
+| First five security assessments | 21 Oct 2020 | [in-toto, OPA, SPIFFE/SPIRE, Harbor, Keycloak](https://github.com/cncf/sig-security/issues/167) |
+| First Tech Leads  | 25 Feb 2020 | [TOC approval](https://lists.cncf.io/g/cncf-toc/topic/71341283#4198) |
```
Comment (lumjjb) [inline roadmap.md:45]:
```suggestion
| First Tech Leads  | 25 Feb 2020 | [TOC approves](https://lists.cncf.io/g/cncf-toc/topic/71341283#4198) [@lumjjb](https://github.com/lumjjb) [@TheFoxAtWork](https://github.com/TheFoxAtWork)  [@JustinCappos](https://github.com/JustinCappos) |
```

File: roadmap.md [hunk 9]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
...
+| Security Assessment Intake Process | 7 Jan 2020 | [intake process and prioritization](https://github.com/cncf/sig-security/pull/296) |
```
Comment (lumjjb) [inline roadmap.md:46]:
```suggestion
| Security Assessment intake process | 7 Jan 2020 | [Intake process and prioritization](https://github.com/cncf/sig-security/pull/296) |
```

File: roadmap.md [hunk 10]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
...
+| First Cloud Native Security Day | 19 Nov 2019 | [event](https://events19.linuxfoundation.org/events/cloud-native-security-day-2019/) organized by [@mfdii and @TheFoxAtWork](https://github.com/cncf/sig-security/issues/209) |
```
Comment (lumjjb) [inline roadmap.md:47]:
```suggestion
| First Cloud Native Security Day | 19 Nov 2019 | [Event](https://events19.linuxfoundation.org/events/cloud-native-security-day-2019/) organized by [@mfdii and @TheFoxAtWork](https://github.com/cncf/sig-security/issues/209) |
```

File: roadmap.md [hunk 11]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
...
+| Software supply chain catalog  | 14 Nov 2019 | [catalog](https://github.com/cncf/sig-security/pull/284) |
```
Comment (lumjjb) [inline roadmap.md:48]:
```suggestion
| Software supply chain catalog  | 14 Nov 2019 | [Catalog](https://github.com/cncf/sig-security/pull/284) |
```

File: roadmap.md [hunk 12]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
...
+| Personas & Use cases | 23 Sept 2019 | [issue#246](https://github.com/cncf/sig-security/issues/246)
```
Comment (lumjjb) [inline roadmap.md:49]:
```suggestion
| Personas & Use cases | 23 Sept 2019 | [Issue#246](https://github.com/cncf/sig-security/issues/246)
```

File: roadmap.md [hunk 13]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
...
+| Policy formal verification overview | 10 Sept 2019 | [documentation](https://github.com/cncf/sig-security/pull/242)
```
Comment (lumjjb) [inline roadmap.md:50]:
```suggestion
| Policy formal verification overview | 10 Sept 2019 | [Documentation](https://github.com/cncf/sig-security/pull/242)
```

File: roadmap.md [hunk 14]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
...
+| First Security Assessment | May 2019 | in-toto - [issue#166](https://github.com/cncf/sig-security/issues/166)  |
```
Comment (lumjjb) [inline roadmap.md:51]:
```suggestion
| First Security Assessment | May 2019 | In-toto - [issue#166](https://github.com/cncf/sig-security/issues/166)  |
```

File: roadmap.md [hunk 15]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
...
 | Updated Charter and Governance ratified by CNCF TOC |  7 May 2019 | [New repo](https://github.com/cncf/sig-security/tree/master/governance) |
+| First cut security audit guidelines  | 2 May 2019 | [guidelines](https://github.com/cncf/sig-security/pull/125) |
```
Comment (lumjjb) [inline roadmap.md:53]:
```suggestion
| First cut security audit guidelines  | 2 May 2019 | [Guidelines](https://github.com/cncf/sig-security/pull/125) |
```

File: roadmap.md [hunk 16]
```diff
@@ -36,9 +36,24 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
...
 | Moved SAFE WG to CNCF  | 15 Apr 2019 | [Repo rename](https://github.com/cncf/sig-security/pull/148) |
 | CNCF WG proposal | 21 Aug 2018 | [CNCF SIG-Security charter and roles](https://github.com/cncf/toc/pull/146) |
 | Policy WG merged | 10 Aug 2018 | [Merging policy WG](https://github.com/cncf/sig-security/blob/master/policy-wg-merging.md)  |
+| First KubeCon Presentations | 2-4 May 2018 | [intro](https://kccnceu18.sched.com/event/ENw3/safe-wg-intro-jeyappragash-j-j-padmeio-ray-colline-google-any-skill-level) and [deep dive](https://kccnceu18.sched.com/event/ENw5/safe-wg-deep-dive-ray-colline-google-intermediate-skill-level) |
```
Comment (lumjjb) [inline roadmap.md:57]:
```suggestion
| First KubeCon Presentations | 2-4 May 2018 | [Intro](https://kccnceu18.sched.com/event/ENw3/safe-wg-intro-jeyappragash-j-j-padmeio-ray-colline-google-any-skill-level) and [deep dive](https://kccnceu18.sched.com/event/ENw5/safe-wg-deep-dive-ray-colline-google-intermediate-skill-level) |
```

File: roadmap.md [hunk 17]
```diff
@@ -36,9 +36,25 @@ Note:  SIG-Security was rebranded from SAFE working group. The below roadmap inc
 
 |   Milestone  | Date | Action
 | --- | --- | --- |
+| First Community Translation | 27 Feb 2021 | [Chinese translation of Whitepaper](https://github.com/cncf/sig-security/pull/471) |
+| Security Assessments => Reviews | 23 Feb 2021 | Retrospective resulted in [process updates](https://github.com/cncf/sig-security/pull/488) |
+| APAC meetings start | 1 Feb 2021 | [Regular meeting time added to README](https://github.com/cncf/sig-security/pull/518)
+| Expanded to 5 Tech Leads | 13 Jan 2021 | [TOC Approves](https://lists.cncf.io/g/cncf-toc/topic/79052801#5599) [@ashutosh-narkar](https://github.com/ashutosh-narkar), [@achetal01](https://github.com/achetal01), [@anvega](https://github.com/anvega) |
+| Cloud Native Security Whitepaper v1 | 18 Nov 2020 | [Markdown source and images in repo](https://github.com/cncf/sig-security/pull/452) |
+| First five security assessments | 21 Oct 2020 | [in-toto, OPA, SPIFFE/SPIRE, Harbor, Keycloak](https://github.com/cncf/sig-security/issues/167) |
```
Comment (lumjjb) [inline roadmap.md:44]:
```suggestion
| First five security assessments | 21 Oct 2020 | [In-toto, OPA, SPIFFE/SPIRE, Harbor, Keycloak](https://github.com/cncf/sig-security/issues/167) |
```

**Review by lumjjb — APPROVED** [review]
LGTM

**Pull request merged**